ptypes/any.go

```diff
@@ -47,9 +47,6 @@ func MarshalAny(m proto.Message) (*anypb.Any, error) {
 	case DynamicAny:
 		m = dm.Message
 	case *DynamicAny:
-		if dm == nil {
-			return nil, proto.ErrNil
-		}
```

Comment:
Thanks for the contribution.

The ptypes package is deprecated. While it exists in its current, frozen state to support existing programs that may depend on it, we are no longer making or accepting changes to it.

I am also, unsure that this code is actually useless. As this code would panic with a nil-deref otherwise:
```
	var m *ptypes.DynamicAny
	ptytes.MarshalAny(m)
̀ ``